libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -169,6 +170,16 @@ string IRGenerator::generate(
 	for (VariableDeclaration const* var: ContractType(_contract).immutableVariables())
 		m_context.registerImmutableVariable(*var);
 
+	string useSrcMap;
+	size_t i = 0;
+	for (auto&& [name, index]: m_context.sourceIndices())
+	{
+		useSrcMap += to_string(index) + ":\"" + name + "\"";
+		if (i++ + 1 != m_context.sourceIndices().size())
```

Comment:
joinHumanReadable?

Reply:
I thought about using that but it didn't seem easy to include the quotes as well

Reply:
```
map<unsigned, string> invertedSourceIndices = invertMap(m_context.sourceIndices());
	string useSrc = joinHumanReadable(
		invertedSourceIndices |
		ranges::views::transform([](auto&& _s) { return to_string(_s.first) + ":" + escapeAndQuoteString(_s.second); })
	);
```
should work here, right?